wordpress-mobile/WordPress-iOS
# iOS 14 widget fixes for unconfigured launch URL and text wrapping

This PR fixes a couple of small issues with the iOS 14 today widget:

* The unconfigured widgets for logged out users appeared to be using the configured widgets `widgetURL` values for some reason (presuming this is a bug). This would result in the app launching, failing to find a site, and bouncing the user back to WordPress in Safari. Setting a `widgetURL` of `nil` for this widget appears to just launch the app as we'd expect.
* Medium widget title text could wrap to two lines, which didn't fit within the widget and ended up with a cropped title label. This PR sets a line limit of 1 for this widget.

Before:

![Image from iOS (1)](https://user-images.githubusercontent.com/4780/107061497-2f56ac80-67d0-11eb-9c87-b53d0b10d174.jpg)

After:

<img width="363" alt="Screenshot 2021-02-05 at 16 17 37" src="https://user-images.githubusercontent.com/4780/107061522-3382ca00-67d0-11eb-9d78-6f9b1440f297.png">

cc @jkmassel 

**To test:**

* Build and run a fresh install of the app
* Before logging in, add a widget
* Tap the widget and ensure the app launches but you're not taken to Safari
* Log in and add a medium widget, and configure it to show a site with a long title
* Ensure the site title doesn't wrap

PR submission checklist:
